cmd/system-shutdown/system-shutdown.c

```
@@ -56,6 +62,8 @@ int main(int argc, char *argv[])
 	if (mkdir("/writable", 0755) < 0) {
 		die("cannot create directory /writable");
 	}
+	// We are reading a file from /run and need to do this before unmounting
+	sc_read_reboot_arg(reboot_arg, sizeof reboot_arg);
```

This is ignoring the error code.